Dockerfile

```diff
@@ -48,8 +54,6 @@ rm -rf Slides/.git
 RUN git clone https://github.com/MaRDI4NFDI/WikibaseImport.git WikibaseImport &&\
 rm -rf WikibaseImport/.git
 
-RUN git clone https://github.com/MaRDI4NFDI/mathsearch_extension.git MathSearch &&\
```

Comment:
i would rather have this repository in current deployment to have working state in production until everything is checked on the other repository.

Reply:
Since we have full control over the gerrit MathSearch repo, I don't see any additional benefit to splitting the development over two repos. We can also have a different branch on MathSearch if that should be required at some point.

Reply:
No. As described above the change can be deployed even before it was reviewed. I did that many times to simplify testing for others.

Reply:
Gerrit is not really intuitive to me, the time consumption overhead is definitely a thing at the moment. 

But,- having a branch in Gerrit and referencing this branch here in the wikibase. Then committing all current changes to this branch and reviewing bigger parts from time to time as a feature is ready and pulling them then to the master branch could solve this.